### DISCUSSION TITLE
Update ideas

### DISCUSSION THREAD

**Discussion by invadergrim** [keiwando/toonsquid#417] [Ideas]
things I believe should be added to toonsquid to keep it on top of its competitors and just to make it a more flexible and comfortable app

-layer effects (for the animation layers and possibly for the drawing layers)
*blurring (Gaussian, motion, perspective, etc), chromatic aberration, noise, sharpness, bloom/lens blur, inner/outer glow, etc

-layer meshing and modern rigging (this might be hard to code though)
*and just better movement tools in general, especially for vector layers

-a better airbrush (the one in there is really crunchy for some reason) 

-better camera motion, smoother tweening for the camera 

-better base canvases, ie a canvas preset for TikTok videos (I usually have to turn the canvas and edit in a different app) 

Among other things but these are all that pop into my head, In no way am I saying I won’t continue to use toonsquid without these, they aren’t necessary, just interesting things I’d love to see

**Comment by DWrightAnimation** [comment]
Fixed Onion Skin Toggle and Out of Pegs would be nice features for hand drawn animators.
